app/controllers/api/transformation_mappings_controller.rb

```diff
@@ -10,6 +10,20 @@ def create_resource(_type, _id, data = {})
       raise BadRequestError, "Could not create Transformation Mapping - #{err}"
     end
 
+    def edit_resource(type, id, data)
+      raise "Must specify transformation_mapping_items" unless data["transformation_mapping_items"]
+      transformation_mapping = fetch_transformation_mapping(type, id)
+
+      updated_data = data.except("transformation_mapping_items")
+      transformation_mapping.update_attributes!(updated_data) if data.present?
```

Comment:
shouldn't this be if updated_data.present ?

Reply:
Yes, correct.

I have fixed that.